### PR TITLE
detail page integrated with api

### PR DESCRIPTION
**Pull request by readwarn**

### CODE REVIEW

**Comment by kayleb01** [comment]
@Didier-ds  please review

File: frontend/src/pages/Details.vue [hunk 1]
```diff
@@ -1,112 +1,281 @@
...
-      <progress id="progress" class="td-w-full td-mx-auto" :value="percent" max="100"> 32% </progress>
-    </div>
-    </div>
-    <!-- <div class="py-2 description ">
+            <div
+                class="header td-py-4 td-flex td-justify-between td-items-center td-border-b-2"
+            >
+                <div>
+                    <h2 class="td-text-xl td-font-bold td-text-green-500">
+                        <i class="pi pi-calendar td-px-2"></i>Todo 01-{{
```
Comment (kayleb01) [inline frontend/src/pages/Details.vue]: Please remove the Todo 01

Comment (readwarn) [reply frontend/src/pages/Details.vue]: done

**Comment by kayleb01** [comment]
please create another PR

**Pull request closed without merging**